BOM/build.gradle

```diff
@@ -0,0 +1,20 @@
+apply plugin: "io.spring.dependency-management"
+
+description = "Bills of Materials (BOM) for Discord4J packages"
+
+compileJava.enabled = false
+javadoc.enabled = false
+tasks.withType(Jar) {
+    enabled false
+}
+test.enabled = false
+
+dependencyManagement {
+    dependencies {
+        project.rootProject.subprojects.each {
```

Comment:
Annotate to this [comment](https://github.com/Discord4J/Discord4J/pull/449#issuecomment-423048652)